selfdrive/car/toyota/carcontroller.py

```diff
@@ -32,9 +33,11 @@ def __init__(self, dbc_name, CP, VM):
     self.alert_active = False
     self.last_standstill = False
     self.standstill_req = False
-
     self.steer_rate_limited = False
 
+    self.use_interceptor = CP.enableGasInterceptor
+    self.pedal_hyst_gap = 3. * CV.MPH_TO_MS
```

Comment:
Does this need to be an attribute. A global constant sounds more appropriate (e.g. `PEDAL_HYST_GAP`).

Reply:
That sounds appropriate. I was thinking about moving them into CarControllerParams, but I think we want to keep that class more unified between makes